mailscanner/checklogin.php

```diff
@@ -68,6 +68,24 @@
     }
 }
 
+if (
```

Comment:
this `if` should be chained with the one on line 53, using an `elseif`, and `else` on line 60 should also set `$_SESSION['user_imap'] = false;`